### DISCUSSION TITLE
KDJ_MACD_Close not in strategies

### DISCUSSION THREAD

**Discussion by panyuleo** [billpwchan/futu_algo#22] [Q&A]
Nice code, Bill.

I couldn't locate the KDJ_MACD_Close class under the strategies library

**Comment by billpwchan** [comment]
Haha you spotted that too. This is a private trading strategy that I'm currently testing with my paper trade account. I'll implement dynamic instantiation soon so that it will only create objects in the strategies folder. Feel free to help me do that if you want!

**Reply by panyuleo** [reply]
Cool. I am still digesting your code. Thinking an alternative way to do backtesting using K_DAY data instead of K_1M, which provides stronger buy/sell decision with different indicators.

**Reply by billpwchan** [reply]
For trading in K_DAY, that should be easy by only changing the subscription in trading_engine.py. For backtesting in K_DAY, that would require some work cuz i didn't write functions to retrieve all the K_DAY data. In general, you can either use the one from yFinance or from futu, which only supports up to 10 years historical daily data.
